src/TaskList.svelte

```diff
@@ -0,0 +1,87 @@
+<script>
+  import { filteredTasks, tasks } from "./stores";
+
+  import TaskListItem from "./TaskListItem.svelte";
+
+  export let alertsOn;
+
+  // function to mark a task as complete, or return it to the array
+  function completeTask(event) {
+    const taskToComplete = event.detail;
+
+    const compliments = [
+      "Nice going!",
+      "Nice job!",
+      "Look at you go!",
+      "You did it!",
+      "Take a bow!",
+      "Wow!",
+      "Woohoo!",
+      "Time for a Mai Tai! 🍹",
+    ];
+    const index = Math.floor(Math.random() * compliments.length);
```

Comment:
You were using `let` for the `compliments` and `index` variables. You should only use `let` if the variable is going to be re-assigned. Since it only gets set once, use `const`.